### PR TITLE
[Testcontainers] Build fixes

### PR DESCRIPTION
**Pull request by Naros**
* Correctly resolve debezium/connect image
* Use Jupiter annotations rather than JUnit4 annotations
* Use Java 17

### CODE REVIEW

File: testcontainers/src/test/java/io/debezium/examples/testcontainers/DebeziumContainerTest.java [hunk 1]
```diff
@@ -48,17 +49,17 @@ public class DebeziumContainerTest {
     private static KafkaContainer kafkaContainer = new KafkaContainer()
             .withNetwork(network);
 
-    public static PostgreSQLContainer<?> postgresContainer = new PostgreSQLContainer<>("debezium/postgres:11")
+    public static PostgreSQLContainer<?> postgresContainer = new PostgreSQLContainer<>(DockerImageName.parse("debezium/postgres:11").asCompatibleSubstituteFor("postgres"))
             .withNetwork(network)
             .withNetworkAliases("postgres");
 
-    public static DebeziumContainer debeziumContainer = new DebeziumContainer(System.getProperty("debeziumVersion"))
+    public static DebeziumContainer debeziumContainer = new DebeziumContainer("debezium/connect:" + System.getProperty("debeziumVersion"))
```
Comment (gunnarmorling) [inline testcontainers/src/test/java/io/debezium/examples/testcontainers/DebeziumContainerTest.java:56]: Where's the value for that property coming from? Can't we just use `DebeziumContainer.latestStable()`, as in other tests?

Comment (Naros) [reply testcontainers/src/test/java/io/debezium/examples/testcontainers/DebeziumContainerTest.java:56]: The value comes from the surefire plugin's configuration to pass `version.debezium` from the pom.

Comment (gunnarmorling) [reply testcontainers/src/test/java/io/debezium/examples/testcontainers/DebeziumContainerTest.java:56]: Ok, but this makes it harder to run this test in an IDE. I'd suggest to use `latestStable()` for ease of use.

Comment (Naros) [reply testcontainers/src/test/java/io/debezium/examples/testcontainers/DebeziumContainerTest.java]: Done

**Comment by gunnarmorling** [comment]
Thx, @Naros!

**Pull request merged**